### PR TITLE
Add first draft of monitoring doc

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Just a first draft of the whats and whys of monitoring.

I wouldn't mind having at trying to split each type of monitoring in to what and why, but first testing the waters to see if this is useful.

### CODE REVIEW

File: docs/monitoring-strategy.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Monitoring things that are important to us allows us to move forward quickly and confidently.
+
+Monitoring should:
+* Clearly let us know if we are succeeding or failing
+* Let us know if we have any faults that need attending (alerting)
+
+## What do we monitor?
+
+### <abbr title="key result">KR</abbr>s]
```
Comment (alexwlchan) [inline docs/monitoring-strategy.md]: trailing square bracket

File: docs/monitoring-strategy.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+__Links__
+* [Wellcome Collection dashboard](https://dash.wellcomecollection.org/)
+
+### [Health](http://ghp.wellcomecollection.org/speedtracker/)
+Alerting is key to health metrics.
+
+Having metrics allow us to release fast and often by letting us know as
+soon as possible that our product is unhealthy. A healthy system is a quiet system.
+
+we should also be able to diagnose where repeated unhealthy spots are to
```
Comment (alexwlchan) [inline docs/monitoring-strategy.md]: /s/we/We

File: docs/monitoring-strategy.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+retrospectively go back and put more permanent solutions
```
Comment (alexwlchan) [inline docs/monitoring-strategy.md]: missing full stop

File: docs/monitoring-strategy.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Having metrics allow us to release fast and often by letting us know as
+soon as possible that our product is unhealthy. A healthy system is a quiet system.
+
+we should also be able to diagnose where repeated unhealthy spots are to
+retrospectively go back and put more permanent solutions 
+
+__Links__
+* [Client side error reporting (Sentry)](https://sentry.io/wellcome/wellcomecollection/dashboard/)
+
+This will unlimately mean more time on new features rather than maintaing <abbr title="business as usual">BAU</abbr>
```
Comment (alexwlchan) [inline docs/monitoring-strategy.md]: /s/unlimately/ultimately

File: docs/monitoring-strategy.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+### Performance
+Performance of our platform and the components it's comprised of are key to user satisfaction.
+
+This could be the speed at which the platform delivers content to a user, or the metrics of
+whether a specific component is being used.
+
+These metrics should allow us to improve or fix areas we deam worth keeping, or culling
```
Comment (alexwlchan) [inline docs/monitoring-strategy.md]: /s/deam/deem

**Comment by jamesgorrie** [comment]
@alexwlchan I've really got to wokr on ym seplling.

**Pull request merged**